### ISSUE TITLE
Payment Intent ID, Charge ID and/or Payment Method ID - consistency / simplification?

### ISSUE THREAD

**Issue by allendav** [Automattic/woocommerce-payments#540]
@LevinMedia - on the order we show the payment intent ID (pi_...), but on the transaction detail we show the charge ID and payment method ID

I think I would change the transaction summary to show the same ID as the order (i.e. the payment intent ID) and move the charge ID to the method box, or at least include the payment intent ID in the method box. What do you think?

**Comment by dwainm** [comment]
When updating the URL we should also update the gateways get_transaction_url method as added here:

https://github.com/Automattic/woocommerce-payments/pull/837

**Comment by a-danae** [comment]
I've opened a PR for the simple text replacement updates. Using the Payment Intent ID in the URL for the Transaction Detail screen is pending. I went deep there but we got new priorities, and what I have is probably too raw to be of use, so leaving this issue unassigned again.

Btw, maybe updating that URL deserves its own issue? It requires its own route in the plugin and updates on how we fetch some data to populate that screen. Not super complicated but maybe unrelated enough from the original issue description to be on its own.
